### ISSUE TITLE
Preloaded Lesson content not visible

### ISSUE THREAD

**Issue by L-Amstutz** [rachelproject/rachelplus#74]
I used the USB Recovery tool to update several RACHEL Plus using method 2, and then cloned the 500G HDD from a properly configured unit.  
According to the device admin page, it used "RACHEL Installer 1-2-16_9.1.1 and the Content Shell v2.2.2
All the wiki and KA-Lite contents in RACHEL seem to be working fine.  However, I just noticed that the RACHEL Content Hub Lesson planner "pre-loaded" items do not show up when logged in as "teacher" to the Library.  However, looking directly on the CAP at /media/preloaded folder, I do see all the teacher editions of textbooks, etc. files are present in the folder (owned by root:root with rwxr--r--  ). Also I found our local "Uploaded" content was listed in the /media/uploaded  folder but it too does not show up in the library.  If I use the teacher GUI interface to upload the local content again, it then shows up in the Content Hub Library, but then the folder on the server ends up with TWO copies of each document (the latest with an appended "1" in the name). I can easily delete and re-add our own content, but not the preloaded content from the GUI.
So it appears the Content Hub Lesson planner is not recognizing pre-existing content on a cloned HDD.  Is there an index file somewhere that might be pointing the wrong place?  How can I get the "pre-loaded" content to show up for teachers and students?

**Comment by needlestack** [comment]
Yes, we've noticed this before: putting files in the /media/preloaded or /media/uploaded folders is not enough -- the system Intel provides only recognizes items that have been uploaded manually through their interface. I don't think we have a solution to this other than to clone from a drive that had the files uploaded manually.

**Comment by L-Amstutz** [comment]
I now have a solution to this, thanks to Bun Tan from Intel.  The preloaded and uploaded content index is maintained in a dbsqlite file that points to the /media/preloaded or /media/uploaded partitions.  So even if you add new content directly to those folders on the HDD, the Content Hub Lesson Planner won't recognize it.  I had added some content to the HDD after cloning, which is what broke the link. The key to fixing my situation was to copy the database file from the eMMC of the original source RACHEL onto the other RACHELs with the cloned HDD, and then relaunch the browser, and PRESTO, all of the preloaded content and uploaded content appear.  The location of this database file is:
/srv/easyconnect/db.sqlite3
I made a backup of the original db file just in case, but as long as the cloned RACHELS  have the same files in the media folders, simply copying the db.sqlite3 file lets it find the content and all works fine.
